src/features/advanced/wf-workforce-filters/wf-workforce-filters.tsx

```
@@ -0,0 +1,82 @@
+        // The first row has a <th> with colspan=2 that messes up the element index.
+        const startingColumn = isHeader ? 5 : 6;
+        for (let i = 0; i < workforceTypes.length; i++) {
+          const workforceType = workforceTypes[i];
+          const column = row.children[startingColumn + i];
+          const isVisible = filters.value.find(x => x.workforce === workforceType)?.value ?? true;
+          column.classList.toggle(css.hidden, !isVisible);
+        }
+
+        const materialLabel = _$(row, C.ColoredIcon.label);
+        if (materialLabel) {
+          const isHidden = !(visibleMaterials.value?.includes(materialLabel.textContent!) ?? true);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)